### DISCUSSION TITLE
LLM and embeddings

### DISCUSSION THREAD

**Discussion by thebrahman** [mudler/LocalAI#1063] [General]
is it possible to run an LLM and embedding model simultaneously from the same docker container?

for example:
![image](https://github.com/go-skynet/LocalAI/assets/8326480/023c35be-6837-44b1-be03-99812e04b902)

**Comment by ianpurton** [comment]
Yes. I'm currently running both in one container.

So when I run

`curl http://localhost:8080/v1/models`

You can see I have 2 models.

```
{"object":"list","data":[{"id":"text-embedding-ada-002","object":"model"},{"id":"ggml-gpt4all-j","object":"model"}]}
```

Then when calling the embeddings API I set the model to `text-embedding-ada-002`

**Comment by BradKML** [comment]
Wait is that part of the UX of LocalAI or some other software?
